src/hooks/useValidation/useValidation.js

```diff
@@ -81,6 +83,22 @@ const useValidation = (context = {}) => {
     }, {});
   };
 
+  // remove field 001 error related to missing field for Bib records during create and derive,
+  // as this field is system generated and expected to be empty.
+  const removeError001MissingField = useCallback(formattedBEValidation => {
+    if (!formattedBEValidation[MISSING_FIELD_ID]) {
+      return formattedBEValidation;
+    }
+
+    return {
+      ...formattedBEValidation,
+      [MISSING_FIELD_ID]: formattedBEValidation[MISSING_FIELD_ID].filter(error => !(
+        context.marcType === MARC_TYPES.BIB
+        && [QUICK_MARC_ACTIONS.CREATE, QUICK_MARC_ACTIONS.DERIVE].includes(context.action)

```

Comment:
We can check that it's not a Bib record or if it's Edit earlier and also return all errors. So inside the filter we only need to check if it's a `001` field